pivy-importer/src/main/groovy/com/linkedin/python/importer/deps/WheelsDownloader.groovy

```diff
@@ -25,18 +27,27 @@ class WheelsDownloader extends DependencyDownloader {
     static final String BINARY_DIST_PACKAGE_TYPE = "bdist_wheel"
     static final String BINARY_DIST_ORG = "wheel"
 
-    WheelsDownloader(String project, File ivyRepoRoot, boolean lenient) {
-        super(project, ivyRepoRoot, lenient)
+    WheelsDownloader(String project, File ivyRepoRoot, DependencySubstitution dependencySubstitution,
+                     boolean latestVersions, boolean allowPreReleases, boolean lenient) {
+        super(project, ivyRepoRoot, lenient, latestVersions, allowPreReleases, dependencySubstitution)
     }
 
     @Override
     def downloadDependency(String dep) {
         log.info("Pulling in $dep")
         def (String name, String version, String classifier) = dep.split(":")
 
-        def projectDetails = cache.getDetails(name)
+        def projectDetails = cache.getDetails(name, lenient)
+        // project name is illegal, which means we can't find any information about this project on PyPI
```

Comment:
Maybe log this message

Reply:
We already have it in the error message, see PypiApiCache.groovy